### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by arnim279**
there we go
reference: [sol sanctum](https://discord.gg/ian) #projects

### CODE REVIEW

**Comment by ibx34** [comment]
lgtm. congrats.

**Review by ibx34 — COMMENTED** [review]
moomail IS pog.

**Review by ibx34 — COMMENTED** [review]
oops

File: README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # *MooMail*
 
+```
+ ____________________________
+< Moomail looks kind of pog. >
```
Comment (splatterxl) [inline README.md:5]:
```suggestion
< Moomail looks really pog. >
```

**Review by splatterxl — CHANGES_REQUESTED** [review]
statement is not positive enough.

File: README.md (hunk 1 above)
Comment (arnim279) [reply README.md:5]: I agree but I think one funny commit is enough (unless @ibx34 is fine with it)

**Pull request merged**